Gemfile

```diff
@@ -25,7 +25,9 @@ gem 'json', :git => 'https://github.com/segiddins/json.git', :branch => 'seg-1.7
 
 group :development do
   cp_gem 'claide',                'CLAide'
-  cp_gem 'cocoapods-core',        'Core'
+  # cp_gem 'cocoapods-core',        'Core'
+  # FIXME: pointing to PR branch for CI
+  gem 'cocoapods-core', :git => 'https://github.com/jkap/Core.git', :branch => 'jkap/app-host-app-spec'
```

Comment:
This merged yet?

Reply:
nah, will merge it once this PR is approved